tests/foreman/ui/test_errata.py

```diff
@@ -1157,59 +1154,110 @@ def test_positive_content_host_search_type(session, erratatype_vm):
 
 
 @pytest.mark.tier3
+@pytest.mark.rhel_ver_match('8')
 @pytest.mark.parametrize(
-    'module_repos_collection_with_setup',
-    [
-        {
-            'distro': 'rhel7',
-            'SatelliteToolsRepository': {},
-            'RHELAnsibleEngineRepository': {'cdn': True},
-            'YumRepository': {'url': settings.repos.yum_9.url},
-        }
-    ],
+    'registered_contenthost',
+    [[CUSTOM_REPO_URL]],
     indirect=True,
 )
-def test_positive_show_count_on_content_host_page(
-    session, module_org_with_parameter, erratatype_vm
-):
-    """Available errata count displayed in Content hosts page
+def test_positive_show_count_on_host_pages(session, module_org, registered_contenthost):
+    """Available errata by type displayed in New Host>Errata page,
+        and expected count by type in Legacy>Content hosts page.
 
     :id: 8575e282-d56e-41dc-80dd-f5f6224417cb
 
     :Setup:
 
-        1. Errata synced on satellite server.
-        2. Some content hosts are present.
+        1. Errata synced on satellite server from custom repository.
+        2. Registered host, subscribed to promoted CVV, with repo synced to appliable custom packages and erratum.
+
+    :steps:
 
-    :steps: Go to Hosts -> Content Hosts.
+        1. Go to Hosts -> All Hosts, and Legacy ContentHost -> Hosts.
+        2. None of the erratum are installable.
+        3. Install all outdated applicable packages via yum.
+        4. Recalculate errata applicablity for host.
+        5. All of the erratum are now installable, on both pages from step 1.
 
-    :expectedresults: The available errata count is displayed.
+    :expectedresults:
+        The available errata count is displayed and updates.
+        Displayed erratum match between the two content host pages.
 
     :BZ: 1484044, 1775427
 
     :customerscenario: true
     """
-    vm = erratatype_vm
+    vm = registered_contenthost
     hostname = vm.hostname
+    assert vm.subscribed
+    assert vm.applicable_errata_count == 0
+
     with session:
+        session.organization.select(org_name=module_org.name)
         session.location.select(loc_name=DEFAULT_LOC)
+        assert vm.execute('subscription-manager repos').status == 0
```

Comment:
I would move this to before the session starts

Reply:
Oh are you doing this to update everything on the host?

Reply:
Yes I had this in session in case there are some app errata present on reg host but not updated, or in case `subscription-manager repos` command is failing which can cause an ambiguous failure later in the test.

For purpose of this case though, I think it is fine to put the command before session starts. 
The only one needed for session would be the command after errata install.